src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -380,20 +380,21 @@ private static bool BuildTargetFrameworkNode(PackageSpec packageSpec, KeyValuePa
             return true;
         }
 
-        private static NuGetFramework GetImports(JObject properties)
+        private static List<NuGetFramework> GetImports(JObject properties)
         {
-            NuGetFramework framework = null;
+            List<NuGetFramework> framework = null;
 
             var importsProperty = properties["imports"];
 
             if (importsProperty != null)
```

Comment:
Use TryGetStringEnumerable here, take a look at "type" as an example, it also allows 1 or Many